### PR TITLE
Added navigation info component

### PR DESCRIPTION
**Pull request by ShairYousuf**
Added the navigation info component with flex display property, future adjustment of css property may be required to be compatible with other components that will be added in the future.

### CODE REVIEW

**Comment by ShairYousuf** [comment]
The UI after adding the navigation info component 
image

File: OnLand/ControlLaptop/angular-UI/src/app/components/navigation-info/navigation-info.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="navigation-info">
+    <H2>Navigation</H2>
+    <p>Longitude: 210</p>
+    <p>Latitude: 40</p>
+    <p>Heading: 10</p>
+    <p>Depth: -1.3</p>
+    <p>Tunnel Length: 20</p>
```
Comment (yeeteing) [inline OnLand/ControlLaptop/angular-UI/src/app/components/navigation-info/navigation-info.component.html:7]: In this case, I think we could try using table tag instead. https://www.w3schools.com/html/html_tables.asp

**Review by yeeteing — COMMENTED** [review]
nice job! 
Just some small changes~

**Comment by ShairYousuf** [comment]
I put the navigation info component into the dashboard component and implemented the table structure for the navigation info.
After edit the page looks like this:
image

**Review by yeeteing — APPROVED** [review]
this is good! Thanks for the work!

**Pull request merged**